primitive/float64_31.ml

```diff
@@ -4,7 +4,7 @@
 (* <O___,, * (see version control and CREDITS file for authors & dates) *)
 (*   \VV/  **************************************************************)
 (*    //   *    This file is distributed under the terms of the         *)
-(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *                     MIT License                            *)
```

Comment:
Are you sure you got agreements of all contributors to relicense that file? (personally, I only agreed about uint63{,_31,_63}.ml{,i}, not that I would disagree though). Same for all float and array files.

Reply:
Ah sorry you're right, I only asked about uint63... Should I do another round of emails, or just ping everyone here?